src/main/java/com/tinyurl/tinyurl/controller/TinyUrlController.java

```diff
@@ -0,0 +1,15 @@
+package com.tinyurl.tinyurl.controller;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+
+public class TinyUrlController
+{
+    @GetMapping("/greetings")
```

Comment:
Doesn't match the issue specification [Issue](https://github.com/rohitnandi12/TinyUrl/issues/1)